src/check/constrain/unify/finished.rs

```diff
@@ -1,9 +1,12 @@
 use crate::check::ast::pos_name::PosNameMap;
 use crate::check::context::{Context, LookupClass};
+use crate::check::context::clss::COLLECTION;
 use crate::check::name::{Empty, Name, Union};
 use crate::check::result::TypeResult;
 use crate::common::position::Position;
 
+const IGNORED_NAMES: [&str; 1] = [COLLECTION];
```

Comment:
Add documentation explaining that IGNORED_NAMES are names which Python does not recognize.
This does unfortunately also make `check` a bit less language agnostic.